pkg/client/client.go

```diff
@@ -93,6 +94,46 @@ func (p ProviderFetchSummary) HasErrors() bool {
 	return false
 }
 
+func (p ProviderFetchSummary) Attributes() []attribute.KeyValue {
```

Comment:
Maybe you should allow conversion of Provider into a map, and create a generic function to convert map -> attribute.KeyValue perhaps?

Reply:
@roneli done